test/markup/maxima/example.txt

```diff
@@ -0,0 +1,79 @@
+/* Maxima computer algebra system */
+
+/* symbolic constants */
+
+[true, false, unknown, inf, minf, ind,
+ und, %e, %i, %pi, %phi, %gamma];
+
+/* programming keywords */
+
+if a then b elseif c then d else f;
+for x:1 thru 10 step 2 do print(x);
+for z:-2 while z < 0 do print(z);
+for m:0 unless m > 10 do print(m);
+for x in [1, 2, 3] do print(x);
+foo and bar or not baz;
+
+/* built-in variables */
+
+[_, __, %, %%, linel, simp, dispflag,
+ stringdisp, lispdisp, %edispflag];
+
+/* built-in functions */
+
+[sin, cosh, exp, atan2, sqrt, log, struve_h,
+ sublist_indices, read_array];
+
+/* user-defined symbols */
+
+[foo, ?bar, baz%, quux_mumble_blurf];
+
+/* symbols using Unicode characters */
+
+[Љ, Щ, щ, Ӄ, ЩЩЩ, ӃӃЉЉщ];
+
+/* different kinds of integers */
+
+ibase : 18 $
+[0, 1234, 1234., 0abcdefgh];
+reset (ibase) $
+
+/* floats and bigfloats */
+
+[0.0, 123.45, .54321,
+ 0e0, .32e10, 123e45, 12.34e56, 123e-4, 234e+5,
+      543.32e1, 4.32e-1, 3.21e+0,
+ 0E0, .32E10, 123E45, 12.34E56, 123E-4, 234E+5,
+      543.32E1, 4.32E-1, 3.21E+0,
+ 0b0, .32b10, 123b45, 12.34b56, 123b-4, 234b+5,
+      543.32b1, 4.32b-1, 3.21b+0,
+ 0B0, .32B10, 123B45, 12.34B56, 123B-4, 234B+5,
+      543.32B1, 4.32B-1, 3.21B+0];
+
+/* strings */
+
+s1 : "\"now\" is";
+s2 : "the 'time' for all good men";
+print (s1, s2, "to come to the aid",
+  "of their country");
+
+/* expressions */
+
+foo (x, y, z) :=
+  if x > 1 + y
+    then z - x*y
+  elseif y <= 100!
+    then x/(y + z)^2
+  else z - y . x . y;
+
+display2d : false $
+foo(%pi, %e, bessel_j(1, u));
+is (% > 0);
+(sin(1) - 1, 2*%%, 1 - %%);
+print ("most recent output is", %);
+
+blurf[foo](a) := a^foo - foo^a $
+blurf[2];
+is (blurf[2] = lambda([a], a^2 - 2^a));
+
+blurf[%pi](%e) - matrix([%phi])^^2;
```

Comment:
Uhm, let me rephrase that. When the demo snippet gets shortened, it does make sense to keep various markup as markup tests but they should test separate concepts (like the one for numbers below), not be a blanket test of the whole language. It tends to break on every maintenance change and hence stops being a good test.